met-web/src/components/engagement/form/EngagementWidgets/WidgetsBlock.tsx

```diff
@@ -33,18 +39,19 @@ const WidgetsBlock = () => {
         handleWidgetDrawerOpen(true);
     };
 
-    const moveWidget = useCallback((dragIndex: number, hoverIndex: number) => {
-        setTempWidgets((prevWidgets: Widget[]) => {
+    const moveWidget = (dragIndex: number, hoverIndex: number) => {
```

Comment:
Not sure what was the reason to use a callback function here, but I had to remove it to have access to fixedWidgets